FA/fa/Finite_Automata.py

```diff
@@ -0,0 +1,96 @@
+import sys
+from termcolor import colored, cprint
+
+class Finite_Automata:
```

Comment:
This class seems to be tailor made for one specific example; could've been a bit more general, although that is easily solvable

Reply:
It seems to but it should work with other ones too, I tried to make some changes to the fa.in and it worked accordingly  😄